machine-learning/tutorials/TaxiFarePrediction/Program.cs

```diff
@@ -31,22 +25,7 @@ static void Main(string[] args)
             // </Snippet3>
 
             // <Snippet4>
-            _textLoader = mlContext.Data.CreateTextLoader(new TextLoader.Arguments()
-            {
-                Separators = new[] { ',' },
-                HasHeader = true,
-                Column = new[]
-                            {
-                                new TextLoader.Column("VendorId", DataKind.Text, 0),
-                                new TextLoader.Column("RateCode", DataKind.Text, 1),
-                                new TextLoader.Column("PassengerCount", DataKind.R4, 2),
-                                new TextLoader.Column("TripTime", DataKind.R4, 3),
-                                new TextLoader.Column("TripDistance", DataKind.R4, 4),
-                                new TextLoader.Column("PaymentType", DataKind.Text, 5),
-                                new TextLoader.Column("FareAmount", DataKind.R4, 6)
-                            }
-            }
-            );
+            //This code has been removed in v0.11.
```

Comment:
Please check this as the code is been removed according ML.Net 0.11

Reply:
Can you remove snippet4 please? Otherwise looks good and I'll merge

Reply:
@JRAlexander Ok I will remove <snippet4> and push changes.